docs/pages/faq.mdx

```diff
@@ -33,20 +33,25 @@ section of the Admin Manual.
 
 ### Can individual nodes create reverse tunnels to a proxy server without creating a new cluster?
 
-This was a popular customer [request](https://github.com/gravitational/teleport/issues/803) that was added in Teleport version 4.0.
-Once you've upgraded your Teleport Cluster, change the node config option `--auth-server` to point to web proxy address (this would be `public_addr` and `web_listen_addr`
-in file configuration). For more information, see [Adding Nodes to the Cluster](./setup/admin/adding-nodes.mdx).
+This was a popular customer
+[request](https://github.com/gravitational/teleport/issues/803) that was added
+in Teleport version 4.0. Once you've upgraded your Teleport Cluster, change the
+node config option `--auth-server` to point to web proxy address (this would be
```

Comment:
```suggestion
`--auth-server` flag when running the `teleport` daemon on an agent to point to the Proxy Service address (this would be
```
Suggestion to make this a bit more precise and use the Proxy Service product name